app/assets/javascripts/utils/date.js

```diff
@@ -1,7 +1,21 @@
-import moment from 'moment';
+import dayjs from 'dayjs';
 
-const isValid = date => moment(date, moment.ISO_8601).isValid();
+const isISO8601 = (date) => {
```

Comment:
This is quite dodgy, but I guess you copied it directly from `moment`. It's fine by now, I guess :sweat_smile: